src/React.Sample.Webpack.CoreMvc/Startup.cs

```diff
@@ -57,11 +52,18 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
 			app.UseStaticFiles();
 
+#if NETCOREAPP3_0
+			app.UseEndpoints(endpoints =>
+			{
+				endpoints.MapControllerRoute("default", "{controller=Home}/{action=Index}/{id?}");
```

Comment:
Probably need to add the `comments/page-{page}` route too?

Reply:
Haha yep fixed this on the plane, forgot to push it